ArraysHomework/ArraysHomework.cs

```diff
@@ -122,6 +167,8 @@ public void Lists()
 
             // Swap the last and first value of this list.
             List<string> swapMyValues = new List<string> { "first", "middle", "last" };
+            
+            swapMyValues.Reverse();
```

Comment:
What if you didn't know how long the list was? If there are more than three elements, then `Reverse` will make chanes we don't want.